src/edgeDetection.h

```diff
@@ -11,13 +11,16 @@
 #include "imageProcessor.h"
 
 
-class edgeDetection: public imageProcessor{
+class edgeDetection:   public imageProcessor{
+   
 public:
     edgeDetection() = default;
+    int threshold= 255;
     void newFrame(frame newFrame);
+    void updateThreshold(int value);
 
 private:
     //add any other methods here
-    void enhanceEdge(frame); //edge detection
+    void enhanceEdge(frame, int threshold); //edge detection
```

Comment:
Don't include threshold in here, it's a member of the class so is already accessible in this scope